src/wp-includes/functions.php

```diff
@@ -8818,7 +8818,15 @@ function clean_dirsize_cache( $path ) {
  * @return bool True if required version is compatible or empty, false if not.
  */
 function is_wp_version_compatible( $required ) {
-	global $wp_version;
```

Comment:
Remove `@global` docblock for `$wp_version` and add docblock for `_wp_tests_wp_version`

Reply:
Done in [857128f](https://github.com/WordPress/wordpress-develop/pull/7104/commits/857128fc26530f2b1405518b6b07ba976ea90c15).